index.js

```diff
@@ -42,10 +42,13 @@ class JestReportPortal {
         let suiteName = testResult.testResults[0].ancestorTitles[0];
 
         this._startSuite(suiteName);
-
         testResult.testResults.forEach(t => {
-            this._startTest(t.title);
-            this._finishTest(t);
+            for (let i = 0; i < t.invocations; i++) {
+                const isRetried = t.invocations === 1 ? null : { retry: true };
```

Comment:
`isRetried` sounds like a boolean variable. I think it should be boolean. No need to wrap it in an object here, you can do this later.